src/Services/Air/AirParser.js

```diff
@@ -531,6 +520,55 @@ const airGetTicket = function (obj) {
   return getTicketFromEtr.call(this, etr, obj);
 };
 
+function airGetTicketsErrorHandler(rsp) {
+  let errorInfo;
+  let code;
+  try {
+    errorInfo = (
```

Comment:
What will happen later on if the condition returns no data? Or it's impossible to have no data returned from the response in this case? If it's possible, maybe it's better to prevent any possible errors that could be occurred on L531

Reply:
Then code will fail and RequestRuntimeError.UnhandledError will be thrown. It's ok here.

Reply:
Got it, thank you.